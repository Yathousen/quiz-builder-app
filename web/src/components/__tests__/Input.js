import React from 'react';
import renderer from 'react-test-renderer';
import { Input } from '../';

it('Input', () => {
  expect(renderer.create(<Input name='Test Name' value='Test Value' />).toJSON()).toMatchSnapshot();
});

it('Disabled Input', () => {
  expect(renderer.create(<Input name='Test Name' value='Test Value' disabled />).toJSON()).toMatchSnapshot();
});
