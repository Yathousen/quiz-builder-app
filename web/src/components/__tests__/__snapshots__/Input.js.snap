// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disabled Input 1`] = `
<div
  className="mt-4"
>
  <label
    className="block mb-2 text-sm font-medium text-gray-600 dark:text-gray-200"
    htmlFor="Test Name"
  >
    Test Name
  </label>
  <input
    className="block w-full px-4 py-2 text-gray-700 bg-white border rounded-md dark:bg-gray-800 dark:text-gray-300 dark:border-gray-600 focus:border-blue-400 focus:ring-opacity-40 dark:focus:border-blue-300 focus:outline-none focus:ring focus:ring-blue-300 disabled:bg-slate-200"
    disabled={true}
    id="Test Name"
    onChange={[Function]}
    value="Test Value"
  />
</div>
`;

exports[`Input 1`] = `
<div
  className="mt-4"
>
  <label
    className="block mb-2 text-sm font-medium text-gray-600 dark:text-gray-200"
    htmlFor="Test Name"
  >
    Test Name
  </label>
  <input
    className="block w-full px-4 py-2 text-gray-700 bg-white border rounded-md dark:bg-gray-800 dark:text-gray-300 dark:border-gray-600 focus:border-blue-400 focus:ring-opacity-40 dark:focus:border-blue-300 focus:outline-none focus:ring focus:ring-blue-300 disabled:bg-slate-200"
    id="Test Name"
    onChange={[Function]}
    value="Test Value"
  />
</div>
`;
